src/main/kotlin/com/hedvig/lookupservice/insurley/service/InsurelyService.kt

```diff
@@ -351,11 +351,22 @@ class InsurelyService(
         }
 
         val apiVersion = insurelySessionRepository.findApiVersionByClientReference(reference)
-        val result = insurelyDataCollectionResultRepository.findByClientReference(reference)
+        val insurelyDataCollectionResult: InsurleyDataCollectionResult
+        try {
+            insurelyDataCollectionResult = insurelyDataCollectionResultRepository.findByClientReference(reference)
```

Comment:
So java to intruduce the val outside of the try catch, we live in the kotlin age now :wink: 
```suggestion
        val insurelyDataCollectionResult = try {
            insurelyDataCollectionResultRepository.findByClientReference(reference)
```

Reply:
Correct, i will fix it